docs/data/routes/docs/nextjs/getting-started/walkthrough-dotnetnew/en.md

```diff
@@ -3,4 +3,160 @@ name: walkthrough-dotnetnew
-# Walkthrough: with `dotnet new`
\ No newline at end of file
+# Walkthrough: Using the .NET starter template for JSS Next.js projects
+
+Sitecore provides a Getting Started template for Sitecore developers that want to try out  Sitecore JSS with [Sitecore Containers](https://containers.doc.sitecore.com/), the [Sitecore Next.js SDK](https://jss.sitecore.com/), and [Sitecore Content Serialization](https://doc.sitecore.com/developers/100/developer-tools/en/sitecore-content-serialization.html). 
+
+For simplicity, this solution does not implement Sitecore Helix conventions for solution architecture. As you begin building your Sitecore solution, you should review [Sitecore Helix](https://helix.sitecore.net/) and the [Sitecore Helix Examples](https://sitecore.github.io/Helix.Examples/) for guidance on implementing a modular solution architecture.
+
+The template includes: 
+
+* A `docker-compose` environment for a Sitecore XP0 topology with a Next.js rendering host.
+
+ \> The included `docker-compose.yml` is a stock XP0 environment from the Sitecore Container Support Package. All changes/additions for this solution are included in the `docker-compose.override.yml`.
+
+* Scripted invocation of `jss create` and `jss import` to initialize a Next.js application.
+
+* Sitecore Content Serialization configuration.
+
+* An MSBuild project for deploying configuration and code into the Sitecore Content Management role. (see `src\platform`).
+
+## Support
+
+The template output as provided is supported by Sitecore. Once changed or amended, the solution becomes a custom implementation and is subject to limitations as defined in Sitecore's [scope of support](https://kb.sitecore.net/articles/463549#ScopeOfSupport).
+
+## Create a solution from the Getting Started template
+
+To create a solution from the Getting Started template, you must complete the following procedures:
+
+- [Install the prerequisites](#)
+- [Install the template]()
+- [Create the MyProject solution]()
+
+### Install the prerequisites
+
+Before you can install the template and create a solution, you must make sure that you have these software components installed on your workstation:
+
+* [NodeJs 14.x](https://nodejs.org/) (we recommend using the latest LTS release.)
+
+* [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) (check your installed version with the `dotnet --version` command)
+
+* [.NET Framework 4.8 SDK ](https://dotnet.microsoft.com/download/dotnet-framework/net48) (see the [Microsoft procedure for checking .NET Framework versions](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed))
+
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+
+* [Docker for Windows](https://docs.docker.com/docker-for-windows/install/), with Windows Containers enabled
+
+See the [Sitecore Containers](https://containers.doc.sitecore.com/) documentation for more information on system requirements.
```

Comment:
The redirect works for this URL but the Paligo direct link is now
https://doc.sitecore.com/developers/100/developer-tools/en/containers-in-sitecore-development.html